problemsets-mat223/linearalbegra.tex

```diff
@@ -1038,24 +1045,81 @@ \section*{Lines and Planes}
 		\item Write $\ell$ in vector form. Is vector form of $\ell$ unique?
-		\item Write $L$ in vector form. 
+			\begin{solution}
+				$\vec x = t \mat{1 \\ -2} + \mat{0 \\ 3}$
+
+				The vector form is not unique, as any scalar multiple of the 
+				direction vector will work in place of $\mat{1 \\ -2}$, as will
+				any other point the line passes through work in place of 
+				$\mat{0 \\3}$. For example, another point on the line is 
+				$\mat{1 \\ 1}$, so the following vector equation describes the 
+				same line: $\vec x = t \mat{-4 \\ 8} + \mat{1 \\ 1}$.
+			\end{solution}
+		\item Write $L$ in vector form.
+			\begin{solution}[inline]
+				$\vec x = t\mat{1 \\ -2 \\ -2} + \mat{0 \\ 3 \\ 3}$
+			\end{solution}
 		\item Find another vector form for $L$ where both ``$\vec d$'' and
 			``$\vec p$'' are different from before.
+			\begin{solution}
+				Again, any multiple of the direction vector will work for 
+				$\vec d$, as will any other point on the line work for 
+				$\vec p$. For example, this equation describes the same 
+				line: $\vec x = t \mat{-3 \\ 6 \\ 6} + \mat{1 \\ 1 \\ 1}$.
+			\end{solution}
 	\end{parts}
 
 	\question
 	Let $A$, $B$, and $C$ be given in vector form by 
-	\[\overbrace{\vec x=t\mat{1\\2\\3}+\mat{0\\0\\1}}^{\displaystyle A}
+	\[
+	\overbrace{\vec x=t\mat{1\\2\\3}+\mat{0\\0\\1}}^{\displaystyle A}
 	\qquad \overbrace{\vec x=t\mat{-1\\1\\1}+\mat{-1\\1\\2}}^{\displaystyle B}
 	\qquad \overbrace{\vec x=t\mat{2\\-1\\1}+\mat{1\\1\\1}}^{\displaystyle C}.
 	\]
 	\begin{parts}
 		\item Do the lines $A$ and $B$ intersect? Justify your conclusion.
+			\begin{solution}
+				Yes. In this case this is quite easy to see since 
+				$\mat{0 \\ 0 \\ 1}$ is a point on line $A$, and this same point
+				 is on line $B$ (for the value $t = -1$).
+			\end{solution}
 		\item Do the lines $A$ and $C$ intersect? Justify your conclusion.
+			\begin{solution}
+				No. In order for them to intersect, there would have to exist 
+				two real numbers $a$ and $c$ such that when substituded for $t$ 
+				in the equations of $A$ and $C$, respectively, the resulting 
+				vectors would equal. In other words, we would have to have:
+				\[
+					a\mat{1\\2\\3}+\mat{0\\0\\1} = c\mat{2\\-1\\1}+\mat{1\\1\\1}.
+				\]
+				This amounts to showing that the following system of equations 
+				has no solutions:
+				\begin{gather*}
+					a = 2c + 1 \\
+					2a = -c + 1 \\
+					3a + 1 = c + 1
+				\end{gather*}
+				The third equation tells us that $c = 3a$, which when substituted 
+				into the first equation forces $a = -\tfrac{1}{5}$ and therefore
+				$c = -\tfrac{3}{5}$. These two numbers don't satisfy the second
+				equation, however.
+			\end{solution}
 		\item Let $\vec p\neq \vec q$ and suppose
 			$X$ has vector form $\vec x=t\vec d+\vec p$ and $Y$ has
 			vector form $\vec x=t\vec d+\vec q$. Is it possible
 			that $X$ and $Y$ intersect?
+			\begin{solution}
+				Yes. This can happen for example if $\vec q$ is a point, other 
```

Comment:
How about something like: "Yes. If $\vec q=\vec p+a\vec d$ for $a\neq 0$, then ..." This way we avoid saying "the vector is a point".

Reply:
Yeah. It's yucky...in a vector space there are no points.

In this class we say that every vector corresponds to the point it points to from the origin and every point corresponds to a vector from the origin to that point, so we will use the interchangeably. That's said on day 1 or 2 and it is mostly ignored from there on out.

Reply:
Gotcha. 

There were a couple of other related things I noticed. I saw that you ask them for  "vectors in a plane", for example. It's because of that and a few similar things that I figured I could be more loose with the distinction.